docs/backend/setup/database-migrations.md

```diff
@@ -101,7 +101,7 @@ When you are writing the code for your new migration, there are a few things you
 This section covers what it's like to add a new table, though some information is helpful when editing existing 
 tables/models as well.
 
-For the examples in this section, we'll be adding a new model called `Cat`, with a corresponding table called `cats`.
+For the examples in this section, we'll be adding a new model called `Pet`, with a corresponding table called `pets`.
```

Comment:
The changes to this file are primarily to support the changes Pearl and I talked about in https://github.com/transcom/mymove-docs/pull/137#discussion_r896084859

These aren't quite necessary, but they do make it so that these models are the same across the docs and add a few examples of things to this page that weren't available with the previous version of the model